activity.py

```diff
@@ -47,7 +47,7 @@ def build_toolbar(self):
         toolbar_box.toolbar.insert(separator, -1)
         separator.show()
 
-        button = ToolButton('speaker-muted-100')
+        button = ToolButton(icon_name='speaker-muted-100')
```

Comment:
What is this change for?  If it is needed, then we should also change sugargame/test/TestActivity.py for the same reason.

Reply:
the change was made by @srevinsaju to fix this error. However, I reviewed it again and changed it as it was in `TestActivity.py`. working perfectly.

Reply:
Yup, we can drop that argument, positional argument corresponds to the kwargs. I am not sure why I did that, but tested it now
@ayushnawal Can you edit my commit and revert the changes at that line only?

Reply:
@srevinsaju I have already pushed a commit with other changes also as @quozl suggested. pr is complete now I guess :grin: